src/components/MetricsPane/MetricsPane.jsx

```diff
@@ -152,57 +154,62 @@ const MetricsPane = ({
+          <MetricCardH3>Surveys</MetricCardH3>
+        </MetricsCard>
+        <MetricsCard>
+          {isDesktopWidth ? (
+            <InlineOnDesktopMetricWrapper>
+              <span>{numUniqueCountries.toLocaleString()}</span>{' '}
+              <MetricCardH3>{countryLabel}</MetricCardH3>
+            </InlineOnDesktopMetricWrapper>
+          ) : (
+            <>
+              <MetricCardPBig>{numUniqueCountries.toLocaleString()}</MetricCardPBig>
+              <MetricCardH3>{countryLabel}</MetricCardH3>
+            </>
+          )}
+        </MetricsCard>
         {isDesktopWidth ? (
-          <InlineOnDesktopMetricWrapper>
-            <span>{numUniqueCountries.toLocaleString()}</span>{' '}
-            <MetricCardH3>{countryLabel}</MetricCardH3>
-          </InlineOnDesktopMetricWrapper>
+          <MultipleMetricCardsRow>{transectAndProjectCountCards}</MultipleMetricCardsRow>
         ) : (
-          <>
-            <MetricCardPBig>{numUniqueCountries.toLocaleString()}</MetricCardPBig>
-            <MetricCardH3>{countryLabel}</MetricCardH3>
-          </>
+          transectAndProjectCountCards
         )}
-      </MetricsCard>
-      {isDesktopWidth ? (
-        <MultipleMetricCardsRow>{transectAndProjectCountCards}</MultipleMetricCardsRow>
-      ) : (
-        transectAndProjectCountCards
-      )}
 
-      {isDesktopWidth ? <MetricsCard>{calculateMetrics.yearRange}</MetricsCard> : null}
-      <MetricsPaneChartTabs
-        id="hard-coral-cover"
-        aggregatePanelContent={<AggregateHardCoralCover />}
-        timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
-      />
-      <MetricsPaneChartTabs
-        id="fish-biomass"
-        aggregatePanelContent={<AggregateFishBiomass />}
-        timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
-      />
-      <MetricsPaneChartTabs
-        id="Bleaching"
-        aggregatePanelContent={<AggregateBleaching />}
-        timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
-      />
-      <MetricsPaneChartTabs
-        id="habitat-complexity"
-        aggregatePanelContent={<AggregateHabitatComplexity />}
-        timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
-      />
-    </SummarizedMetrics>
+        {isDesktopWidth ? <MetricsCard>{calculateMetrics.yearRange}</MetricsCard> : null}
+      </SummarizedMetrics>
+      <ChartsWrapper $showMobileExpandedMetricsPane={showMobileExpandedMetricsPane}>
+        <MetricsPaneChartTabs
+          id="hard-coral-cover"
+          aggregatePanelContent={<AggregateHardCoralCover />}
+          timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
+        />
+        <MetricsPaneChartTabs
+          id="fish-biomass"
+          aggregatePanelContent={<AggregateFishBiomass />}
+          timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
```

Comment:
could simplify the place holder:

`const ComingSoonPlaceholder = () => <div style={{ height: '270px' }}>Coming soon</div>`